### DISCUSSION TITLE
How to customize status-bar?

### DISCUSSION THREAD

**Discussion by tony-sol** [zellij-org/zellij#2644] [Q&A] [answered]
I'm trying to move to zellij from tmux and want to acheive same UI and UX

I have couple plugins, provides current `username@hostname port` with ssh support on `status-left`, mode indicator (WAIT|COPY|SYNC|TMUX) and clock with date on `status-right` and using native parameters:
```
window-status-separator "|"
window-status-format " #{window_index}:#{window_name} (#{window_panes}) "
window-status-current-format " #{window_name} (#{window_panes}) "
pane-border-format " #{pane_index}:#{pane_tty} #{pane_current_command} "
```
which gives me this appearance:

<img width="682" alt="tmux" src="https://github.com/zellij-org/zellij/assets/33671815/3c85db89-dad5-4888-8b4b-4940bab6bd03">

Is there a way to achieve same via default `status-bar` or `compact-bar` plugins or/and with custom layout?
Maybe there is any plugins repo/list/compilation, like awesome-zellij-plugins where i may find? ( google says nothing :c )

**Comment by dj95** [comment, marked as answer]
Hi @Tony-Sol ,

I wrote a statusbar plugin, which let's you customize the look and feel like in tmux.

It's still in development, but works for me with some smaller bugs.
But I'm working on it, since I want to use it in my daily driver setup.

https://github.com/dj95/zjstatus

Edit: Fixed known issues
